src/commands/du.rs

```diff
@@ -180,17 +185,22 @@ impl FileInfo {
         let path = path.into();
         let name = path.to_string_lossy().to_string();
         let m = if deref {
-            std::fs::metadata(path)
+            std::fs::metadata(&path)
         } else {
-            std::fs::symlink_metadata(path)
+            std::fs::symlink_metadata(&path)
         };
 
         match m {
-            Ok(d) => Ok(FileInfo {
-                name,
-                size: d.len(),
-                tag,
-            }),
+            Ok(d) => {
+                let block_size = file_real_size_fast(path, &d).ok();
+
+                Ok(FileInfo {
+                    name,
+                    blocks: block_size,
```

Comment:
If this is to be a *block count* rather than a *physical file size*, this would need to be divided by the block size.  Traditional `du`'s allow this to be configured with `-B`, defaulting to 512.

Otherwise this is arguably misleading, it's not a number of blocks, it's a size in bytes.

Reply:
Oh, sorry. For some reason I got it in my head that it was blocks.